SimTracker/TrackerMaterialAnalysis/test/trackingMaterialAnalyser_ForHFNosePhaseII.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env cmsRun
```

Comment:
Hello @mariadalfonso. Quick question, which process did you follow to validate this PR? Running `cmsRun SimTracker/TrackerMaterialAnalysis/test/trackingMaterialAnalyser_ForHFNosePhaseII.py` is not working on the latest IB:

```
Begin processing the 1st record. Run 1, Event 1, LumiSection 1 on stream 0 at 06-Mar-2020 22:21:41.361 CET
TrackingMaterialProducer::update(const BeginOfJob*): WARNING: selected volume "HFNose" not found in geometry 
Begin processing the 2nd record. Run 1, Event 2, LumiSection 1 on stream 0 at 06-Mar-2020 22:21:59.976 CET
Begin processing the 3rd record. Run 1, Event 3, LumiSection 1 on stream 0 at 06-Mar-2020 22:21:59.981 CET
Begin processing the 4th record. Run 1, Event 4, LumiSection 1 on stream 0 at 06-Mar-2020 22:21:59.993 CET
Begin processing the 5th record. Run 1, Event 5, LumiSection 1 on stream 0 at 06-Mar-2020 22:21:59.998 CET
Begin processing the 6th record. Run 1, Event 6, LumiSection 1 on stream 0 at 06-Mar-2020 22:22:00.004 CET
Begin processing the 7th record. Run 1, Event 7, LumiSection 1 on stream 0 at 06-Mar-2020 22:22:00.012 CET


A fatal system signal has occurred: segmentation violation
The following is the call stack containing the origin of the signal.
```
We are migrating this package (`SimTracker/TrackerMaterialAnalysis`) to DD4hep, if you can help us providing information regarding the validation process followed on this PR would be great!